settings.gradle

```diff
@@ -202,7 +200,7 @@ project(':ballerina-lang:bool').projectDir = file('langlib/lang.boolean')
 project(':ballerina-lang:runtime').projectDir = file('langlib/lang.runtime')
 project(':ballerina-lang:regexp').projectDir = file('langlib/lang.regexp')
 project(':ballerina-lang-test').projectDir = file('langlib/lang.test')
-project(':ballerina-langlib:test').projectDir = file('langlib/langlib-test')
+project(':ballerina-lang:test').projectDir = file('langlib/langlib-test')
```

Comment:
@Shadow-Devil Can you please keep this as is? langlib is used to be the commonly known name for langlib modules